homebrew/liftoff.rb

```diff
@@ -3,7 +3,7 @@
 class Liftoff < Formula
   homepage 'https://github.com/thoughtbot/liftoff'
   url 'http://thoughtbot.github.io/liftoff/Liftoff-__VERSION__.tar.gz'
-  sha1 '__SHA__'
+  sha256 '__SHA__'
```

Comment:
Ignore this. I'll look at this soon.

> On May 20, 2016, at 19:22, Hound <notifications@github.com> wrote:
> 
> In homebrew/liftoff.rb:
> 
> > @@ -3,7 +3,7 @@
> >  class Liftoff < Formula
> >    homepage 'https://github.com/thoughtbot/liftoff'
> >    url 'http://thoughtbot.github.io/liftoff/Liftoff-__VERSION__.tar.gz'
> > -  sha1 '__SHA__'
> > +  sha256 '__SHA__'
> Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>